blog/tag/seo.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: tag
+tag: seo
+permalink: /blog/tag/seo/
+redirect_from:
```

Comment:
One other closing statement that I think would be helpful is to understand the search landscape. Doesn't Google really own 94% at the moment? So SEO really means Google optimization at this point in time? Giving some context to that, perhaps early on in the post would be helpful.

Overall, great!